index.html

```diff
@@ -386,122 +349,104 @@ <h3 id="ruby-character-size-and-character-placement">Ruby character size and cha
 <section>
 <h3 id="placement-of-mono-ruby">Placement of mono-ruby</h3>
 
-<p>Mono-ruby is placed as follows. 
-To align following items to the <a href="#l20200529009">two-step processing method</a> described in 
-<a href="#matters-considered-by-the-placement-rules"></a>, 
-points 1, 2, and 3 are of the first step, and points 4 and 5 are of the second 
-step. 
-</p>
+<p>Mono-ruby is placed as follows.</p>
+<p>To align the following items to the <a href="#l20200529009">two-step processing method</a> described in [[[#matters-considered-by-the-placement-rules]]], 
```

Comment:
should we remove "-" from two-step, since the original state was rewritten without "-"?

Reply:
The hyphen makes it easier to read: it's a two-step process, not two step-processes.  So i prefer to keep it. Unresolve this if you feel strongly.

Reply:
Then, I'd like to propose to modify line 239 to have "-".

Reply:
I've looked at two versions of the document for line 239, but can't see any text where this would make sense. Please provide the text you want to see changed, rather than just a line number (which will anyway be useless for future readers of this comment if other changes are made).

Reply:
Not having heard back over the past hour, i'll proceed to merge this. Please raise against the new version, thanks.